README.md

```diff
@@ -36,15 +36,27 @@ Der Server bietet sowohl für die App als auch für den Bot die notwendigen API
 Der Source-Code und Dokumentation des Servers ist in diesem Repo zu finden.
 
 #### Install and Start
-Voraussetzung ist ein installiertes python 3.8 package
-- `sudo ./manage.sh install` um den server zu installieren
-  (sudo only necessary to install graphviz to display story graph)
-- `./manage.sh start` um den server zu starten
-- für weitere befehle `./manage.sh help` benutzen
+Voraussetzung ist ein Linux System mit installiertem python 3.8
+1. `./manage.sh install` um den server zu installieren
+1. `./manage.sh start` um den server zu starten
+1. für weitere befehle `./manage.sh help` benutzen
+1. siehe Bot repo für Installation und Start des Bots
+1. nach Start von Server und Bot kann die App und damit das Spiel gestartet werden
 
 #### Architektur-Überblick
 Der Server wird durch eine Flask App (`/app`) implementiert. Diese verwaltet verschiedene API-Endpunkte (`/app/routes`). Außerdem wird eine Datenbank verwaltet. Deren ORM wird in `/app/models` implementiert. Der Story-Inhalt und der Story-verwaltende Code liegt unter `/app/story`.  
 
+#### Nutzung ohne App
```

Comment:
We also should explain why the response `"saving images is not yet supported by the database"` is replies upon sending a photo.
This could also be fixed by a better response string

Reply:
right... how about adding `calling '.../finished' will automatically complete the task`?